system-test/urth-system-test-specs.js

```diff
@@ -69,3 +71,64 @@ describe('Widgets System Test', function() {
             .nodeify(done);
     });
 });
+
+args.python != "python2" && describe('Widgets Scala System Test', function() {
```

Comment:
I was trying to use this logic on the R system tests, but noticed an issue.  I think this should be `process.env.PYTHON != "python2" && ...`.  `args.python` is undefined so this will always be `true && ...` and the tests will always run.